packages/core/src/sheets/worksheet.ts

```diff
@@ -86,9 +87,95 @@ export class Worksheet {
         return this._spanModel;
     }
 
+    /**
+     * Get the style of the column.
+     * @param {number} column The column index
+     * @returns {Nullable<IStyleData>|string} The style of the column
+     */
+    getColumnStyle(column: number): Nullable<IStyleData> | string {
+        return this._styles.get(this._snapshot.columnData[column]?.s);
+    }
+
+    getColumnStyleInternal(column: number): Nullable<IStyleData> {
+        const style = this._snapshot.columnData[column]?.s;
+        if (typeof style === 'string') {
+            return this._styles.get(style);
+        }
+        return style;
+    }
+
+    /**
+     * Set the style of the column.
+     * @param {number} column The column index
+     * @param {string|Nullable<IStyleData>} style The style to be set
+     */
+    setColumnStyle(column: number, style: string | Nullable<IStyleData>): void {
+        const columnData = this._snapshot.columnData[column];
+        if (!columnData) {
+            this._snapshot.columnData[column] = { s: style };
+        } else {
+            columnData.s = style;
+        }
+    }
+
+    /**
+     * Get the style of the row.
+     * @param {number} row The row index
+     * @returns {Nullable<IStyleData>} The style of the row
+     */
+    getRowStyle(row: number): Nullable<IStyleData> {
+        return this._styles.get(this._snapshot.rowData[row]?.s);
+    }
+
+    getRowStyleInternal(row: number): Nullable<IStyleData> {
+        const style = this._snapshot.rowData[row]?.s;
+        if (typeof style === 'string') {
+            return this._styles.get(style);
+        }
+        return style;
+    }
+
+    /**
+     * Set the style of the row.
+     * @param {number} row
+     * @param {string|Nullable<IStyleData>} style The style to be set
+     */
+    setRowStyle(row: number, style: string | Nullable<IStyleData>): void {
```

Comment:
Maybe it would be better to move to `RowManager`

Reply:
it's a good idea, I will do it before this PR merge into dev